<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XSLExpressClass</class>
 <widget class="QWidget" name="XSLExpressClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>417</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>XSLExpress</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>380</width>
     <height>398</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="settingsLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Saved Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="savedSettings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveCurrent">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteCurrent">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="filenamesLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Output. Replace:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="replaceThis">
        <property name="text">
         <string>\.xml</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>with</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="replaceWith">
        <property name="text">
         <string>-output.xml</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="xslLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>XSL File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="DropFilenameLineEdit" name="xslFile"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="getParameters">
        <property name="text">
         <string>Load parameters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="getParametersWithDefaults">
        <property name="text">
         <string>Load parameters with default values</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearValues">
        <property name="text">
         <string>Clear values</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLineEdit" name="paramName1"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="paramName2"/>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="paramName3"/>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="paramName4"/>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="paramName5"/>
      </item>
      <item row="1" column="1">
       <widget class="DropFilenameLineEdit" name="paramValue1"/>
      </item>
      <item row="2" column="1">
       <widget class="DropFilenameLineEdit" name="paramValue2"/>
      </item>
      <item row="3" column="1">
       <widget class="DropFilenameLineEdit" name="paramValue3"/>
      </item>
      <item row="4" column="1">
       <widget class="DropFilenameLineEdit" name="paramValue4"/>
      </item>
      <item row="5" column="1">
       <widget class="DropFilenameLineEdit" name="paramValue5"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Parameter Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Parameter Value</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="inputLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="DropFilenameTextEdit" name="inputFiles"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="processLayout">
      <item>
       <widget class="QCheckBox" name="processImmediately">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Process files upon drop</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="process">
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="allowOverwrite">
      <property name="text">
       <string>Overwrite input files if the regular expression does not change them</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DropFilenameLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">dropfilenamelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>DropFilenameTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">dropfilenametextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>savedSettings</tabstop>
  <tabstop>replaceThis</tabstop>
  <tabstop>replaceWith</tabstop>
  <tabstop>xslFile</tabstop>
  <tabstop>paramName1</tabstop>
  <tabstop>paramValue1</tabstop>
  <tabstop>paramName2</tabstop>
  <tabstop>paramValue2</tabstop>
  <tabstop>paramName3</tabstop>
  <tabstop>paramValue3</tabstop>
  <tabstop>paramName4</tabstop>
  <tabstop>paramValue4</tabstop>
  <tabstop>paramName5</tabstop>
  <tabstop>paramValue5</tabstop>
  <tabstop>inputFiles</tabstop>
  <tabstop>processImmediately</tabstop>
  <tabstop>allowOverwrite</tabstop>
  <tabstop>process</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
